mne_bids/write.py

```diff
@@ -1166,10 +1233,32 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
     _participants_json(participants_json_fname, True, verbose)
     _scans_tsv(raw, op.join(kind, bids_fname), scans_fname, overwrite, verbose)
 
-    # TODO: Implement coordystem.json and electrodes.tsv for EEG and  iEEG
+    # TODO: Implement coordystem.json and electrodes.tsv for EEG
     if kind == 'meg' and not emptyroom:
         _coordsystem_json(raw, unit, orient, manufacturer, coordsystem_fname,
-                          overwrite, verbose)
+                          kind, overwrite, verbose)
+    elif kind == 'ieeg':
+        electrodes_fname = make_bids_basename(
+            subject=subject_id, session=session_id, acquisition=acquisition,
+            suffix='electrodes.tsv', prefix=data_path)
+
+        # We only write iEEG electrodes.tsv and accompanying coordsystem.json
+        # if we have an available DigMontage
+        if raw.info['dig'] is not None:
+            coords = _extract_landmarks(raw.info['dig'])
+
+            # Rescale to MNE-Python "head" coord system, which is the
+            # BIDS "ElektaNeuromag" system, and equivalent to BIDS "CapTrak"
```

Comment:
These systems are not BIDS systems per se, similar to how FIFF is not a BIDS system ... BIDS just accepts their usage

```suggestion
            # "ElektaNeuromag" system (equivalent to "CapTrak" system)
```